### PR TITLE
:sparkles: Create SECURITY_CONTACTS and SECURITY.md

### PR DESCRIPTION
**Pull request by clubanderson**
required for CNCF sandbox and for IBM Dev Advocates office

## Related issue(s)

Fixes #

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+
+## Security Announcements
+
+Join the [kubestellar-security-announce](https://groups.google.com/u/1/g/kubestellar-security-announce) group for emails about security and major API announcements.
+
+## Report a Vulnerability
+
+We're extremely grateful for security researchers and users that report vulnerabilities to the KubeStellar Open Source Community. All reports are thoroughly investigated by a set of community volunteers.
+
+You can also email the private [kubestellar-security-announce@googlegroups.com](mailto:kubestellar-security-announce@googlegroups.com) list with the security details and the details expected for [all KubeStellar bug reports](https://github.com/kcp-dev/edge-mc/blob/main/.github/ISSUE_TEMPLATE/bug_report.yaml).
```
Comment (MikeSpreitzer) [inline SECURITY.md:10]: "announce" sounds wrong for the ingress of problem reports, "announce" suggests this is for egress from the maintainers to the public.

File: SECURITY_CONTACTS [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Defined below are the security contacts for this repo.
+#
+# They are the contact point for the Product Security Committee to reach out
+# to for triaging and handling of incoming issues.
+#
+# The below names agree to address security concerns if and when they arise.
+#
+# DO NOT REPORT SECURITY VULNERABILITIES DIRECTLY TO THESE NAMES, FOLLOW THE
+# INSTRUCTIONS AT https://kubernetes.io/security/
```
Comment (MikeSpreitzer) [inline SECURITY_CONTACTS:9]: Too eager copy-and-paste here.

File: SECURITY.md (hunk 1 above)
Comment (clubanderson) [reply SECURITY.md:10]: I used the same naming convention as Kubernetes

File: SECURITY_CONTACTS (hunk 2 above)
Comment (clubanderson) [reply SECURITY_CONTACTS:9]: Thank you

Comment (pdettori) [reply SECURITY_CONTACTS]: /lgtm

File: SECURITY.md (hunk 1 above)
Comment (MikeSpreitzer) [reply SECURITY.md:10]: I think not.  Compare https://kubernetes.io/docs/reference/issues-security/security/#security-announcements with https://kubernetes.io/docs/reference/issues-security/security/#report-a-vulnerability

Comment (clubanderson) [reply SECURITY.md:10]: yes - so here is my thinking on that one - there was a whole bunch of structure around a security response committee (SRC) - which I was not ready to commit to at this time.  So I made it simple, for now.  Aspiration is to get there, when we get a bit more resourced.

Comment (MikeSpreitzer) [reply SECURITY.md:10]: Sure.  I got the simplification.  My only point is that the term "announce" is a confusing choice for an ingress point.

**Pull request merged**